sys/net/application_layer/nanocoap/nanocoap.c

```diff
@@ -810,6 +748,15 @@ size_t coap_opt_put_string(uint8_t *buf, uint16_t lastonum, uint16_t optnum,
     return bufpos - buf;
 }
 
+size_t coap_opt_put_uint(uint8_t *buf, uint16_t lastonum, uint16_t onum,
+                         uint32_t value)
+{
+    uint32_t tmp = value;
```

Comment:
Is a local copy needed here?

Reply:
Good catch. Looks like I can use the address of a parameter passed by value rather than create the tmp variable. I plan to leave in the statement to assign the length to `tmp_len` though, for clarity.

Reply:
OK!